src/gui/gui.pri

```diff
@@ -63,6 +63,7 @@ HEADERS += \
     $$PWD/transferlistsortmodel.h \
     $$PWD/transferlistwidget.h \
     $$PWD/updownratiodialog.h \
+    $$PWD/uithememanager.h \
```

Comment:
@jagannatharjun:  It is still in the wrong place.  These filenames should be listed in **alphabetical order**.  

In the English version of the Latin alphabet, `i` comes before `p`.  

```
$$PWD/updownratiodialog.h \
$$PWD/uithememanager.h \
```

…should be… 

```
$$PWD/uithememanager.h \
$$PWD/updownratiodialog.h \
```